src/controllers/hotels.controller.js

```diff
@@ -100,11 +102,70 @@ class Hotel {
       type,
       description,
       cost,
-      status
+      status,
     });
 
     return Responses.handleSuccess(201, 'Room added successfully', res, room);
   }
+
+  /**
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} responses
+   */
+  async getMostVisitedDestination(req, res) {
+    const page = (req.query.page * 1) || 1;
+    const results = [];
+    let trips = [];
+    const requests = await getAllRequestsHotels();
+
+    if (!requests.length) {
+      return Responses.handleError(404, 'There is no past travel yet.', res);
+    }
+
+    const requestTrips = requests.map(el => el.trips);
+
+    for (let i = 0; i < requestTrips.length; i += 1) {
+      trips = trips.concat(requestTrips[i]);
+    }
```

Comment:
This operation looks to be unnecessary. `requests.map(el => el.trips);` is returning an array of trips. Is there a reason to copy the  `requestTrips` into an empty array which is `trips`. Basically, `requestTrips` and `trips` are to contain the same data.

Reply:
You're right. It was a duplication of code.